src/components/SnackBar/SnackBar.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import MuiSnackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import { SnackBarProps } from './types';
+
+const SnackBar: React.FC<SnackBarProps> = ({
+  message,
+  isOpen,
+  onClickCloseButton,
+}) => {
+  return (
+    <MuiSnackbar
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'left',
+      }}
+      open={isOpen}
+      autoHideDuration={3000}
+      message={<span>{message}</span>}
+      action={[
+        <IconButton
+          key='close'
+          aria-label='close'
+          color='inherit'
+          onClick={onClickCloseButton}>
+          <CloseIcon />
+        </IconButton>,
+      ]}
+    />
+  );
+};
+
+export default SnackBar;
```

Comment:
@lucianojss Regarding the `Snackbar` separate component - seems current implementation doesn't fully cover the whole of its logic. I belive `open/close` state should be included as well. 

To make all the things working under the hood, I came up with HOC solution [here](https://github.com/BuildIt-Poland/buddy-app/blob/615db5db46eaad3bce09799a6f48672cfbcb6d1c/src/decorators/withSnackBar/withSnackBar.tsx)

As a result it should help with keeping all the `Snackbar` logic in one place, facilitate its integration to the screens/components and help with writing more testable code.